src/test/run-make-fulldeps/rustdoc-target-spec-json-path/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../tools.mk
```

Comment:
nit: can you use `include` instead so it gives an error if the file is missing?

Reply:
Sure, I just copied another rustdoc test.